tests/scripts/format.py

```diff
@@ -69,67 +69,6 @@ def main(argv):
 
     my_env = os.environ
 
-    # Download .NET CLI
```

Comment:
I don't care too much about removing all this stuff from format.py, which only the CI uses. But similar code exists for the pmi-asm-diffs py script, and I want to make it possible to use that with just a Python install, so maybe we should think about creating a simple python script that does the dotnet install for that.